velox/functions/lib/QuantileDigest.h

```diff
@@ -1371,6 +1386,77 @@ QuantileDigest<T, Allocator>::getDistributionFunction(
   return cdf;
 }
 
+template <typename T, typename Allocator>
+template <typename ValueFunc>
+std::optional<double> QuantileDigest<T, Allocator>::getTruncatedMean(
+    double lowerQuantileBound,
+    double upperQuantileBound,
+    ValueFunc valueFunc) const {
```

Comment:
What would be the ValueFunc in a real use case? Are we expecting different ValueFunc for different UDFs?

Reply:
In this case the argument and return types are simple so I removed it